src/main/java/com/minecolonies/coremod/blocks/BlockHutWareHouse.java

```diff
@@ -0,0 +1,35 @@
+package com.minecolonies.coremod.blocks;
+
+import com.minecolonies.coremod.tileentities.TileEntityWareHouse;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Hut for the warehouse.
+ * No different from {@link AbstractBlockHut}
+ */
+
+public class BlockHutWareHouse extends AbstractBlockHut
```

Comment:
up to 5 deliverymen can work at 1 warehouse.
WareHouse is a central building, deliveryman is the hut for the dman.
There are further plans for usages for the deliveryman hut.